themes/ddbasic/sass/p2/p2-view.scss

```diff
@@ -42,3 +42,12 @@ tr.draggable {
     }
   }
 }
+
+// ==========================================================================
+// View ding public lists
+// ==========================================================================
+.pane-public-lists {
```

Comment:
This seems to refer to a panel pane. Why is this not in `p2-class-panel.scss`?

Reply:
Has been moved

Reply:
:+1: